src/index.html

```diff
@@ -5,10 +5,53 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img class="header__logo"
+          src="./images/logo.png"
+          alt="moyo-logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+              data-qa="hover">
+              laptops & computers
+            </a>
```

Comment:
visually, the content with "laptops and computers" looks like part of the attributes, so it is better to place the closing corner on a separate line
```suggestion
            <a
              class="nav__link"
              href="#"
              data-qa="hover"
            >
              laptops & computers
            </a>
```

Reply:
+